ops/model.py

```diff
@@ -3640,12 +3642,31 @@ def secret_set(
         args = [id]
         if label is not None:
             args.extend(['--label', label])
+            self._secret_cache[id]['label'] = label
+        elif 'label' in self._secret_cache[id]:
+            args.extend(['--label', self._secret_cache[id]['label']])
         if description is not None:
             args.extend(['--description', description])
+            self._secret_cache[id]['description'] = description
+        elif 'description' in self._secret_cache[id]:
+            args.extend(['--description', self._secret_cache[id]['description']])
```

Comment:
Pls consider adding blank line to separate `if/elif` blocks from independent `if`'s.

Reply:
This method is basically a big bunch of if statements processing the arguments, followed by the bit that does the content and the actual set call. Adding in a blank line between each of the ifs would make for a very broken up chunk of code, and it doesn't seem more readable to me. All the arg checking is done with separate if statements, but is all _logically_ grouped.

Reply:
The code was moved from here anyway :)